.github/scripts/render_card.py

```
@@ -0,0 +1,495 @@
+
+    # Notable versions table
+    rows = "\n".join(
+        f"| {p['name']} | `{p['version']}`"
+        + (f" | `{p['prev']}` → `{p['version']}`" if p.get("changed") and p.get("prev") else " | —")
+        + " |"
+        for p in versions["notable"]
+    )
+
+    # Build lines with no leading whitespace — 4-space indent renders as code in GitHub MD
+    L = [
+        f"<!-- card-light -->",
```

Ruff (0.15.12)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)